### DISCUSSION TITLE
Organisations / labels

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#6126] [Ideas]
I am part of a team of translators, and it would be useful to have this team available on each instance.
Maybe projects want to run their own team, maybe they want multiple ones.
It is getting to be difficult to push quality further without one, and organizing it all takes away from getting there.

Between organizing what goes in, fixing it, and getting translators to do the work, it would be a nice place to go and point to.

The required functionality is:
Inviting people to a team
Requesting addition to projects
Ability to throw out a team from a project I suppose
The statistics (explained below)
Doesn't have to inherit privileges, but it would be nice to have hand-picked reviewers for more than just one project at a time.

With just the default project listing, there is enough opportunity to describe it in the infobox, and add a link to some means of communication. That way people can get into the loop rather than get into the habit of drive-by translation. Right now it is difficult to find out who the contributors to a project is for non-admins, when more info about who does what would be better.

So a project with symbolic links to projects.

My other request that I have been thinking about forever is labels.
I have lost track of how many Android projects there are, but sometimes I remember the license which narrows it down a bit.
If all of these things had labels, and they were organized in a similar fashion (though without owners) it would really help various sections find their audience.

Operating system, packaging type, program type, Non-CoC, CoC, License, Direct editing possible.

Then it becomes easy to have a landing site for all SailfishOS projects, made in C, if that is what people are interested in making their dashboard or links to give away. Maybe you want to find "communication" software by way of discovery that way, and so on.

**Comment by comradekingu** [comment]
I think this could be used to navigate to stuff needing translation too.
I can't remember my system from SecureDrop, but something like:
If the release is tagged with a deadline in the source repo, translation maintainers could set time-frames and it could all chug along automatically

#stringfreeze_deadline_one_week
##stringfreeze_deadline_one_day
#stringfreeze

#deadline_one_week
#deadline_one_day

**Reply by Melmorgan1** [reply]
888
